docs/pipelineruns.md

```diff
@@ -1182,17 +1182,44 @@ For more information, see the [`LimitRange` support in Pipeline](./compute-resou
 
 ### Configuring a failure timeout
 
-You can use the `timeouts` field to set the `PipelineRun's` desired timeout value in minutes.  There are three sub-fields than can be used to specify failures timeout for the entire pipeline, for tasks, and for `finally` tasks.
+You can use the `timeouts` field to set the `PipelineRun's` desired timeout value in minutes.
+There are three sub-fields:
+- `pipeline`: specifies the timeout for the entire PipelineRun. Defaults to to the global configurable default timeout of 60 minutes.
+When `timeouts.pipeline` has elapsed, any running child TaskRuns will be canceled, regardless of whether they are normal Tasks
+or finally Tasks, and the PipelineRun will fail.
```

Comment:
Use code style to highlight that this means the type of task and not just an adverb. Same for the other instances of "finally" in this section.

Reply:
done